protocols/imap/src/test/java/org/apache/james/imap/encode/ListResponseEncoderTest.java

```diff
@@ -50,20 +48,64 @@
     @Before
     public void setUp() throws Exception {
         mockNextEncoder = context.mock(ImapEncoder.class);
-        response = new SearchResponse(IDS, null);
```

Comment:
you no longer test that case ?

Reply:
ok, I saw it after. Yet, it's the kind of fix that should be done in a separated commit